core/modules/utils/dom/popup.js

```diff
@@ -136,7 +142,19 @@ Popup.prototype.show = function(options) {
 			height: options.domNode.offsetHeight
 		};
 	}
-	var popupRect = "(" + rect.left + "," + rect.top + "," + 
+
+	if (options.absolute && options.domNode) {
+		// Walk the offsetParent chain and add the position of the offsetParents to make
```

Comment:
Is there any scope for making a global utility function to aid with creating popup coordinate strings?

Reply:
The newest update to this PR contains a commit that implements a function for that. I like the symmetry between `buildCoordinates` and `parseCoordinates` 👍 Please have a look at my last comment for more details.